src/main/java/no/nav/veilarbperson/controller/v3/PersonV3Controller.java

```diff
@@ -32,10 +34,16 @@ public class PersonV3Controller {
 
     @PostMapping("/hent-person")
     @Operation(summary = "Henter informasjon om en person fra PDL")
-    public PersonV2Data hentPerson(@RequestBody PersonRequest personRequest) {
+    public PersonV2Data hentPerson(@RequestBody PdlRequest pdlRequest) {
```

Comment:
Jeg har snakka litt med Sondre, og han har foreslått at hvert endepunkt egentlig burde ha hver sin request-type, i stedet for at vi har en "PdlRequest" og "PersonRequest", siden de jo henter forskjellige ting. 
Det er litt jobb å fikse det, så jeg skal bare forstå 100 % hvorfor det er hensiktsmessig, hvis noen har noe jeg kan lese eller vil forklare bare skriv :)